content/concept-history-sharding.md

```diff
@@ -30,21 +30,21 @@ The following snippet from an example `rippled.cfg` file shows the configuration
 ```
 [shard_db]
 type=NuDB
-path=/Users/rippled/.rippled/db/shards/nudb
+path=/var/lib/rippled/db/shards/nudb
 max_size_gb=50
 ```
 
 **Tip:** Ripple recommends using NuDB for the shard store (`type=NuDB`). NuDB uses fewer file handles per shard than RocksDB. RocksDB uses memory that scales with the size of data it stores, which may require excessive memory overhead.
 
-**Tip:** While both validator and tracking (or stock) `rippled` servers can be configured to use history shard stores, Ripple recommends adding history sharding only for non-validator nodes to reduce overhead for validator nodes. If you run a validator node and want to manage ledger history using sharding, run a separate `rippled` server with sharding enabled.
+**Tip:** While both validator and tracking (or stock) `rippled` servers can be configured to use history shard stores, Ripple recommends adding history sharding only for non-validator nodes to reduce overhead for validator nodes. If you run a validator and want to manage ledger history using sharding, run a separate `rippled` server with sharding enabled.
 
 For more information, reference the `[shard_db]` example in the [rippled.cfg configuration example](https://github.com/ripple/rippled/blob/master/doc/rippled-example.cfg).
 
 ### Sizing the Shard Store
 Determining a suitable size for the shard store involves careful consideration. The following concerns contribute to determining shard store sizing:
 
 - Although redundant, it is possible to hold full ledger history in the node store and the history shard store
-- Each shard of the node store is 2^14 ledgers (16384), which may not be optimal
+- Each shard of the node store is 2^14 ledgers (16384)
```

Comment:
Also, let's add a comment giving a ballpark of how much disk space and time a ledger shard consists of.

For disk size, let's say:

> A shard occupies 200 MB to 4 GB based on the age of the shard. Older shards are smaller because there was less activity in the XRP Ledger at the time.

As for time, my calculations tell me the oldest ledgers shards occupy about 85 hours' worth of time, and the most recent shards occupy about 16-17 hours. So how about this?

> A shard represents about 10 to 100 hours of time, depending on the rate at which new ledgers are closed. As of early 2018, recent shards tend to represent about 15-20 hours.

Reply:
Great!